### PR TITLE
kubernetes docs update

### PR DESCRIPTION
**Pull request by wasaga**
Split Kubernetes documentation into separate sections:

- Installation (new guide)
- Global Configuration (new guide)
- Ingress Configuration (rewritten)
- CRD Reference

### CODE REVIEW

**Comment by desimone** [comment]
LGTM; if you don't mind could we wait till https://github.com/pomerium/documentation/pull/170 is merged. I'm in conflict merge hell :)

File: content/docs/k8s/configure.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+Integration with your Identity Provider is configured using [`identityProvider`](./reference#identityprovider) parameter. 
+
+### Authenticate endpoint
+
+Each Pomerium installation has a special route that unauthenticated users are redirected to that handles sign-in via your Identity Provider. It is configured via the [`authenticate`](./reference#authenticate) parameter.
+
+The authenticate endpoint DNS address should resolve to an external IP address assigned by your Kubernetes Load Balancer to the `pomerium-proxy` service. If you use `external-dns`, that may be [done automatically](#external-dns).
+
+:::note
+You should not create a separate Ingress resource for the Authenticate URL.
```
Comment (desimone) [inline content/docs/k8s/configure.md:55]:
```suggestion

You should not create a separate Ingress resource for the Authenticate URL.

```

**Review by desimone — COMMENTED** [review]
LGTM pending @ZPain8464 . Can you also run prettier on the file, and add prepended and post-pended newlines to the admonition blocks so prettier doesn't garble the blocks?

https://docusaurus.io/docs/markdown-features/admonitions

**Comment by wasaga** [comment]
> Can you also run prettier on the file

it's always running on save.

**Review by ZPain8464 — APPROVED** [review]

**Comment by ZPain8464** [comment]
This one looks good to me 👍

**Pull request merged**